config/settings.yml

```diff
@@ -902,11 +902,16 @@
     - SuspendVM_Task
 :host_scan:
   :queue_timeout: 20.minutes
-:http_proxy:
-  :host:
-  :password:
-  :port:
-  :user:
+ec2_http_proxy:
```

Comment:
I thought we could namespace the proxies by `ems.ems_type` i.e. `ec2`, `openstack`.

What was the thought of how this is populated… by a form on the app, or by hand by the end user.

If the latter we'd need to make sure they knew how to format it. We could just auto create the fields with empty values, like we do for `http_proxy`